crates/bitwarden-api-api/src/models/update_domains_request_model.rs

```diff
@@ -16,7 +16,7 @@ pub struct UpdateDomainsRequestModel {
         rename = "excludedGlobalEquivalentDomains",
         skip_serializing_if = "Option::is_none"
     )]
-    pub excluded_global_equivalent_domains: Option<Vec<i32>>,
+    pub excluded_global_equivalent_domains: Option<Vec<crate::models::GlobalEquivalentDomainsType>>,
```

Comment:
We should decouple the server response/requests from this enum in the future. No need to be aware of the 70 equivalent domains.